app/web/main.py

```diff
@@ -21,13 +21,22 @@
 
 celery = get_celery()
 
-def app_factory(settings = get_settings()):
+
+def app_factory(settings: Settings = None):
+    # TODO: Create dev, test, and prod versions of settings that do not have
+    # TODO: to be passed in as a parameter
+    if settings is None:
+        settings = get_settings()
```

Comment:
The next step will be to have the `Settings` object setup automatically depending on the `ENV` variable.